blackmouse572/quizler-fe
# Feat/generate invitation link

- Fetch classroom key and generate link based on key
- Quick copy

Test:
1. go to http://localhost:3000/classrooms/2
2. click on link icon
Expected:
- Code was generated
- Link and code is the same
- Can copy on click button
